examples/cli_example_training_ephys.json

```diff
@@ -0,0 +1,45 @@
+{
+  "generator_params": {
+    "pre_post_frame": 30,
+    "steps_per_epoch": 10,
+    "total_samples": -1,
+    "name": "EphysGenerator",
+    "batch_size": 100,
+    "start_frame": 2000,
+    "randomize": 0,
+    "train_path": "/Users/jeromel/Documents/Work documents/Allen Institute/Projects/Deep2P/repos/public/deepinterpolation/sample_data/ephys_tiny_continuous.dat2",
```

Comment:
I assume the point of this file is to show users what the JSON file needs to look like (in which case, it probably doesn't matter that this directory is on your personal machine)?

Reply:
Yes this is the intent